cmd/clean.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/numaproj-labs/numaflow-perfman/util"
+)
+
+// cleanCmd represents the clean command
+var cleanCmd = &cobra.Command{
+	Use:   "clean",
+	Short: "Remove all perfman resources",
+	Long:  "Delete all resources from K8s clusters that perfman created during setup ",
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		// Delete Prometheus Operator
+		if err := util.UninstallRelease(util.PrometheusReleaseName, util.PerfmanNamespace, log); err != nil {
+			return fmt.Errorf("failed to delete Prometheus Operator: %w", err)
+		}
+
+		// Delete Grafana
+		if err := util.UninstallRelease(util.GrafanaReleaseName, util.PerfmanNamespace, log); err != nil {
+			return fmt.Errorf("failed to delete Grafana: %w", err)
+		}
+
+		// Delete pipeline metrics service monitors
```

Comment:
i agree with the part-of=perfman part, however we would still need to delete the resource individually because they have different gvr definitions, i.e. we cannot delete a service monitor and pipeline in the same loop

Reply:
Could you double-check on this? We can use `kubectl delete all -l labelKey=labelValue` to delete all resources no matter the type. It's unlikely that K8 clients don't provide an equivalent...

Reply:
from my understanding kubetctl under the hood is still making requests to the api for each type of resource we have. I was not able to figure out a way that we can just loop over generic resources and delete any of them with a specific label.

Reply:
fair.

Reply:
Then adding the label is not quite helpful. Still good to have it.